src/asttypename.d

```diff
@@ -0,0 +1,112 @@
+/**
+ * Part of the Compiler implementation of the D programming language
+ * Copyright:   Copyright (c) 1999-2017 by The D Language Foundation, All Rights Reserved
+ * Authors:     Stefan Koch
+ * License:     $(LINK2 http://www.boost.org/LICENSE_1_0.txt, Boost License 1.0)
+ * Source:      $(DMDSRC _asttypename.d)
+ */
+
+module ddmd.asttypename;
```

Comment:
Can we go with "ast_typename" instead? I think it's easier to read when the words are separated. Or is too much different from the rest of the code?